webcompat/static/js/lib/issues.js

```diff
@@ -53,6 +53,22 @@ md.linkify.add("#", {
     match.url = "/issues/" + match.url.replace(/^#/, "");
   }
 });
+
+var checkIfReportedURLOpeningA = function(tokens) {
+  // fast-check the first 2 tokens, returning false if we don't match
+  if (tokens[0].markup !== "**" && tokens[1].content !== "URL") {
```

Comment:
not sure to understand this test. We should test:
```
if (tokens[0].markup == "**" && tokens[1].content == "URL") 
```

We want to add an attr `target="_blank"` for the URl reported by the author, isn'it ?